packages/e2e-tests/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:10.16.3
+
```

Comment:
Just to be clear, this is only for running e2e and not for running persistent seeder in production. Right? 

For production node app, we should not run as root user.

Reply:
Actually this is probably going to be used in _production_.

For running e2e tests, we will look to integrate into our current circle ci pipeline instead of using this Dockerfile.